src/components/fundamentals/input-header.tsx

```diff
@@ -0,0 +1,32 @@
+import React from "react"
+import InfoTooltip from "../info-tooltip"
+
+type InputHeaderProps = {
+  label: string
+  required?: boolean
+  withTooltip?: boolean
+  tooltipText?: string
+  tooltipProps?: any
+}
+
+const InputHeader: React.FC<InputHeaderProps> = ({
+  label,
+  required = false,
+  withTooltip = false,
+  tooltipText,
+  tooltipProps,
+}) => {
+  return (
+    <div className="w-full flex pb-1.5 font-semibold text-small leading-xsmall text-grey-50 items-baseline">
```

Comment:
In the global.css file there is a number of text utility classes that you can use instead of writing out the all the classes. In this instance you could add the class `inter-small-semibold`. This would also set the line height to 20px and not 16px as you have done here which I believe is wrong according to the Figma? (Please correct me if I'm wrong)

Reply:
you are right 😄  had the right font size but not line height, it's fixed